docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -1,51 +1,81 @@
 [[getting-started]]
-== Quick start: Add data using Elasticsearch APIs
+== Quick start: Indices, documents, and mappings
 ++++
-<titleabbrev>Basics: Add data using APIs</titleabbrev>
+<titleabbrev>Basics: Indices, documents, and mappings</titleabbrev>
 ++++
 
 In this quick start guide, you'll learn how to do the following tasks:
 
-* Add a small, non-timestamped dataset to {es} using Elasticsearch REST APIs.
-* Run basic searches.
-
-[discrete]
-[[add-data]]
-=== Add data
-
-You add data to {es} as JSON objects called documents.
-{es} stores these
-documents in searchable indices.
-
-[discrete]
-[[add-single-document]]
-==== Add a single document
-
-Submit the following indexing request to add a single document to the
-`books` index.
-The request automatically creates the index.
+* Create an index
+* Add data as documents
+* Use dynamic and explicit mappings of data types
+* Do basic searches
 
 ////
 [source,console]
 ----
 PUT books
+PUT my-explicit-mappings-books
 ----
 // TESTSETUP
 
 [source,console]
 --------------------------------------------------
 DELETE books
+DELETE my-explicit-mappings-books
 --------------------------------------------------
 // TEARDOWN
 
 ////
 
+[discrete]
+[[getting-started-index-creation]]
+=== Create index
+
+Create a new index named `books`:
+
+[source,console]
+----
+PUT /books
+----
+// TEST[skip: index already setup]
+
+List all indices to confirm the `books` index was created:
+
+[source,console]
+----
+GET /_cat/indices?v&h=index,docs.count&s=index:asc
```

Comment:
Do we want to show what the response looks like for this. You have example responses for other examples put not this one. Just thinking out loud.

Reply:
I'll resolve this because we talk about examples in general in another https://github.com/elastic/elasticsearch/pull/113558#discussion_r1778178420